Formula/payara.rb

```diff
@@ -1,13 +1,15 @@
 class Payara < Formula
   desc "Java EE application server forked from GlassFish"
   homepage "https://www.payara.fish"
-  url "https://search.maven.org/remotecontent?filepath=fish/payara/distributions/payara/5.192/payara-5.192.zip"
-  sha256 "272352a4d8a6fd19a0e3e02bde946fb9a860c1206fc6e39a41279a73f43b2995"
-  revision 1
+  url "https://search.maven.org/remotecontent?filepath=fish/payara/distributions/payara/5.2021.4/payara-5.2021.4.zip"
+  sha256 "b080057e4e2e4e7eed09eba67fe4777c2aa4a0eca401ec480d677812b7591521"
 
-  bottle :unneeded
+  livecheck do
+    url "https://search.maven.org/remotecontent?filepath=fish/payara/distributions/payara/"
+    regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
+  end
 
-  depends_on "openjdk@8"
+  depends_on "openjdk@11"
```

Comment:
Trying `openjdk@11` for ARM support.
Docker images are available with Zulu11: https://hub.docker.com/layers/payara/micro/5.2021.4-jdk11/images/sha256-d3c16686cfd2804e9d7b853429f9d950ac5a22ccfc7457dc0371f5443ba59399?context=explore